go-controller/pkg/ovn/ovn.go

```diff
@@ -153,6 +155,12 @@ type Controller struct {
 	// An address set factory that creates address sets
 	addressSetFactory addressset.AddressSetFactory
 
+	// if the required CRDs are in the system enable egressfirewall
+	egressfirewallEnabled bool
+	// holds the dnsInformation for egressfirewalls indexed by dnsName
+	egressfirewallDNSInfo  map[string]*dnsInformation
```

Comment:
Is there any reason why the item is a pointer?

Reply:
other similar objects are pointers so that is what I used